test/alternator/suite.yaml

```diff
@@ -1 +1,23 @@
-type: Run
+type: Python
+pool_size: 4
+extra_scylla_cmdline_options:
+#   - '--alternator-address=8080'
+    - '--alternator-enforce-authorization=1'
+    - '--alternator-write-isolation=always_use_lwt'
+    - '--alternator-streams-time-window-s=0'
+    - '--alternator-timeout-in-ms=30000'
+    - '--alternator-ttl-period-in-seconds=0.5'
+#    - '--alternator_port=8000'
+  # Allow testing experimental features. Following issue #9467, we need
+  # to add here specific experimental features as they are introduced.
+  # We only list here Alternator-specific experimental features - CQL
+  # ones are listed in test/cql-pytest/run.py.
+#    - '--experimental-fa   eatures=alternator-streams'
+run_first:
```

Comment:
Since even the slowest Alternator test files take just a few seconds, is it worthwhile to try to mark which should run first?

Reply:
Please note that for this specific test to NOT take a very long time, Scylla should be configured with a low query_tombstone_page_limit. In test/cql-pytest.py we configure '--query-tombstone-page-limit', '1000'. As far as I can tell, test.py doesn't. Please fix that. I hope it will speed up this test.
Please also check you have   '--num-tokens', '16' - this speeds up CDC tests.

Reply:
query-tombstone-page-limit is added, num_tokens is set to 16 by default config.